lib/zold/remotes.rb

```diff
@@ -224,13 +227,24 @@ def error(host, port = PORT)
       if_present(host, port) { |r| r[:errors] += 1 }
     end
 
+    def unerror(host, port = PORT)
+      raise 'Host can\'t be nil' if host.nil?
+      raise 'Port can\'t be nil' if port.nil?
+      raise 'Port has to be of type Integer' unless port.is_a?(Integer)
```

Comment:
I see those checks on host and port are repeated in many methods of this class, why not moving everything to a method and call this instead every time it is requested?

Reply:
@lastk Normally, you should do this modification in this PR. The reason for that is that while you were at it, you should have noticed that this code was repeated many times in the same class and should have seen this as an opportunity for refactoring. However, if you spent too much time on this issue already, just leave a puzzle and the PDD bot will take care of opening a new issue

Reply:
oh, I see, well I'm still getting used to the process, once I saw that I shouldn't touch any code which is not related to the task... Ok, I can do it in this pr for sure :)